### ISSUE TITLE
!! VSCode marketplace DOES NOT mention telemetry

### ISSUE THREAD

**Issue by ognjenio** [dendronhq/dendron#675]
I only accidentally noticed this because of #671 and noticed Segment initializing. 

I'm quite unhappy about this because the changelog on VSCode marketplace does not mention that telemetry was added. I assume that's the changelog that most people would open, rather then going to the website for every update. 

There is also no mention of it at all on the marketplace page or in the Github readme.

At best this is an oversight, but to me definitely not an insignificant one. At worst it was intentionally left out in order to have fewer people be aware of it. 

e: Honestly, it smells that adding telemetry was marked as a `chore` which the changelog doesn't seem to pick up.

**Comment by kevinslin** [comment]
Hey @ognjenio,

Thanks for raising the ticket. Telemetry was added in last weeks release. 

We dedicated an entire section in our [release notes](https://wiki.dendron.so/notes/82130f1f-521d-42ec-a7da-b76a82f0d39d.html) about this change as well as published the full details of the limited [telemetry](https://wiki.dendron.so/notes/84df871b-9442-42fd-b4c3-0024e35b5f3c.html) we collect, why we do so, and how to opt out.

Not updating the README was an oversight - we actually haven't updated this for quite a while (the last update was 3 months ago). 
As for the changelog, we have an update notification after every upgrade linking to our manually published changelog [here](https://wiki.dendron.so/notes/9bc92432-a24c-492b-b831-4d5378c1692b.html). Unfortunately there doesn't seem a way to link this to what is displayed in VSCode which is why it falls back to the auto-generated changelog. For auto-generation, you can see how we classify work items [here](https://wiki.dendron.so/notes/81da87be-2d4e-47b5-a1d6-c0d647e1ab00.html#commit-style). The reason telemetry was classified as a `chore` is because it is not a direct user facing feature or enhancement. 

Based on your issue, we've taken the following actions:
- updated our README to include a section on telemetry: https://github.com/dendronhq/dendron/blob/master/README.md#L179:L179
- updated our auto-generated changelog to include a link to our published changelog:  https://github.com/dendronhq/dendron/blob/master/packages/plugin-core/CHANGELOG.md#L1:L1

So to sum up, sorry that this came as a surprise - we tried to broadcast this change prominently but it's clear we missed some important channels.

**Comment by kpathakota** [comment]
@ognjenio - thanks for reporting. Based on Kevin's comment, I'm closing this issue but feel free to reopen if there's still more to address here.
